setup.cfg

```diff
@@ -63,8 +63,10 @@ install_requires =
 	lxml
 	Pillow
 	psycopg2-binary
+	pyotp
 	python-dateutil
 	python-magic
+	qrcode
```

Comment:
Since these dependencies are needed on the prod server, it is not sufficient to add them to the Pipfile, we need to add them to setup.cfg as well.